### PR TITLE
docs(topic-rd): update 2.0 migrating guide

### PR DESCRIPTION
**Pull request by TeodorTaushanov**

### CODE REVIEW

File: docs/Migrating to version 2.0 guide.md [hunk 1]
```diff
@@ -188,6 +193,19 @@ Now you should use `pageIndicatorType` instead:
 <ui5-carousel page-indicator-type="Numeric"></ui5-carousel>
 ```
 
+- Properties `items-per-page-s`, `items-per-page-m`, `items-per-page-l` are removed with a single `items-per-page` property and adds additional size `XL`
```
Comment (ilhan007) [inline docs/Migrating to version 2.0 guide.md:196]: replaced, not removed

File: docs/Migrating to version 2.0 guide.md [hunk 2]
```diff
@@ -891,6 +1007,8 @@ now use the `overflowButton` slot:
 </ui5-tabcontainer>
 ```
 
+ - The display CSS property of the component is changed from `inline-block` to `block`.
```
Comment (ilhan007) [inline docs/Migrating to version 2.0 guide.md:1010]: I think we can remove this, it's not a breaking change like API

File: docs/Migrating to version 2.0 guide.md (hunk 1 above)
Comment (TeodorTaushanov) [reply docs/Migrating to version 2.0 guide.md]: Done.

File: docs/Migrating to version 2.0 guide.md (hunk 2 above)
Comment (TeodorTaushanov) [reply docs/Migrating to version 2.0 guide.md]: Done.

File: docs/Migrating to version 2.0 guide.md [hunk 3]
```diff
@@ -188,6 +193,19 @@ Now you should use `pageIndicatorType` instead:
 <ui5-carousel page-indicator-type="Numeric"></ui5-carousel>
 ```
 
+- Properties `items-per-page-s`, `items-per-page-m`, `items-per-page-l` are replaced by a single property `items-per-page` and adds additional size `XL`
```
Comment (BorisDafov) [inline docs/Migrating to version 2.0 guide.md:196]: I suggest:
- Properties `items-per-page-s`, `items-per-page-m`, `items-per-page-l` are replaced by a single property `items-per-page`, which also adds an additional `XL` size

File: docs/Migrating to version 2.0 guide.md [hunk 4]
```diff
@@ -404,6 +424,42 @@ Now you have to use it like:
 <ui5-dialog state="Success"></ui5-dialog>
 ```
 
+- `isOpen` and `close` methods are no longer present. Now `open` property can be used instead.
```
Comment (BorisDafov) [inline docs/Migrating to version 2.0 guide.md:427]: We should restructure the sentence in order to start it with capital letter:
Methods `isOpen` and `close` are no longer present. Now `open` property can be used instead.

File: docs/Migrating to version 2.0 guide.md [hunk 5]
```diff
@@ -404,6 +424,42 @@ Now you have to use it like:
...
+let isOpen = dialog.isOpen();
+dialog.close();
+```
+Now:
+```ts
+let isOpen = dialog.open;
+dialog.open = false;
+```
+
+- `show` method is no longer present. `open` property can be used instead.
```
Comment (BorisDafov) [inline docs/Migrating to version 2.0 guide.md:440]: - Method `show` is no longer present. Use `open` property instead.

File: docs/Migrating to version 2.0 guide.md [hunk 6]
```diff
@@ -404,6 +424,42 @@ Now you have to use it like:
...
+Previously:
+```ts
+dialog.show();
+```
+Now:
+```ts
+dialog.open = true;
+```
+
+- parameter `preventInitialFocus` from method `show` is added as a property.
```
Comment (BorisDafov) [inline docs/Migrating to version 2.0 guide.md:451]: Parameter `preventInitialFocus` from method `show` is added as a property.

File: docs/Migrating to version 2.0 guide.md [hunk 7]
```diff
@@ -670,20 +729,77 @@ Now use `placement` instead:
...
+let isOpen = popover.isOpen();
+popover.close();
+```
+Now:
+```ts
+let isOpen = popover.open;
+popover.open = false;
+```
+
+- `showAt` method is no longer present.
```
Comment (BorisDafov) [inline docs/Migrating to version 2.0 guide.md:745]: Also, if there is a replacement - please list it. For example:
Method `showAt` is no longer present. Use `opener` instead.

File: docs/Migrating to version 2.0 guide.md [hunk 8]
```diff
@@ -670,20 +729,77 @@ Now use `placement` instead:
...
+```ts
+popover.showAt(opener);
+```
+Now:
+```ts
+popover.opener = opener;
+popover.open = true;
+```
+
+- parameter `preventInitialFocus` from method `showAt` is added as a property.
```
Comment (BorisDafov) [inline docs/Migrating to version 2.0 guide.md:757]: Parameter `preventInitialFocus` from method `showAt` is added as a property.

File: docs/Migrating to version 2.0 guide.md (hunk 3 above)
Comment (TeodorTaushanov) [reply docs/Migrating to version 2.0 guide.md]: Done.

File: docs/Migrating to version 2.0 guide.md (hunk 4 above)
Comment (TeodorTaushanov) [reply docs/Migrating to version 2.0 guide.md]: Done.

File: docs/Migrating to version 2.0 guide.md (hunk 6 above)
Comment (TeodorTaushanov) [reply docs/Migrating to version 2.0 guide.md]: Done.

**Review by ilhan007 — APPROVED** [review]

**Review by BorisDafov — COMMENTED** [review]
I approve the PR from UA perspective

**Pull request merged**